### ISSUE TITLE
simple note field (client-side and server-side)

### ISSUE THREAD

**Issue by roycewilliams** [hashtopolis/server#898]
During large-scale loose collaborations, coordinating across many clients, and also coordinating between multiple server admins, is very labor-intensive, especially for keeping track of state/progress/status/quirks 

Two address this, _two_ simple note/description fields would be super helpful, one on each side:

- one that could be initially set by the client (during the data-entry phase, "please enter an optional one-line short description or note, or Enter to leave blank", and then stored in config.jsoin

-  one set/controlled on the server side

Each would be read only on the "other side" - on the server, the client note would be shown as a read-only field, and on the client side, the server note would be shown to the client _at the beginning of each block_.

For little effort, this creates a super minimalist info-sharing channel between client and server, and also allows state keeping to persist across client-only and/or server-only teams.

This capability would also has value even when there's only a single fleet ("this is the box with the flaky second GPU", etc.)

**Comment by zyronix** [comment]
To confirm, you want a info box sent from the agent to the server so on the agent detail pages the server admin sees the description? Something like: 'zyronix's agent: old laptop with sometimes overheating GPU' that you can set during starting of the agent.

What is the usecase for the server sending 'messages' to the agent?

**Comment by s3inlc** [comment]
> This capability would also have value even when there's only a single person managing a fleet ("this is the box with the flaky second GPU", etc.)

Do I understand correctly, you also would see it as a purpose to identifiy an agent? Hashtpolis both offers setting a useful agent name after the creation (e.g. stating what kind of agent it is like 'laptop-xy') and also to set an owner, which then would tell who is responsible for the hardware. Can you explain why this is not enough for your use-case?

**Comment by roycewilliams** [comment]
Yep, totally understood that this is feature overlap. But after directly experiencing the use case of "large-scale loose collaborations" - a hundred different people all showing up simultaneously who all want to help, but have no hashtpolis experience, and being the only server-side administrator during such a rush - the goal here is to have the _option_  to decentralize, and enable the users to do more of the client preparation work. So any "after the creation" steps that would normally be handled by the server admin(s), but that could be easily handled on the client side, would help.

**Comment by s3inlc** [comment]
Same as for the client side attack parameters I would propose that such a feature would need to be activated as a config on the server side, and also not being anything blocking when asking for input, as this would mess with the auto-setup of agents.

**Comment by roycewilliams** [comment]
Understood that it needs to be non-blocking. Since voucher (for example) is something that only blocks if it's missing, perhaps setting the client-comment to an empty string in config.json would be a way to support auto-setup.
